packages/satori-html/src/index.ts

```diff
@@ -77,62 +77,73 @@ interface VNode {
   };
 }
 
-export function html(
-  templates: string | TemplateStringsArray,
-  ...expressions: any[]
-): VNode {
-  const result = __html.call(null, templates, ...expressions);
-  let doc = parse(result.value.trim());
-  inliner(doc);
-  tw(doc);
+export interface SatoriHtmlOptions {
+  tailwind?: boolean;
+}
+
+export function htmlWithOption(options?: SatoriHtmlOptions) {
+  return function html(
+    templates: string | TemplateStringsArray,
+    ...expressions: any[]
+  ): VNode {
+    const result = __html.call(null, templates, ...expressions);
+    let doc = parse(result.value.trim());
+    inliner(doc);
+
+    if (options?.tailwind) {
+      tw(doc);
+    }
```

Comment:
I've only changed this part, otherwise it's just returning the some function as before.